src/tree/assess/scoreAggregators/ReadabilityScoreAggregator.js

```diff
@@ -0,0 +1,167 @@
+import { sum } from "lodash-es";
+import { scoreToRating } from "../../../interpreters";
+
+/* Internal dependencies */
+import { ScoreAggregator } from "../index";
+
+/**
+ * Total number of available readability assessments.
+ *
+ * @type {number}
+ * @const
+ */
+const totalNrOfAssessments = 8;
```

Comment:
Yes. I think the most straight forward way is the way that you describe.
So: a file with all the readability assessments that gets
1. Imported (or defined) and used in the to-be-implemented `constructReadabilityAssessor()` function to make a new `TreeAssessor` for the readability analysis.
2. Gets imported here, with this line changing to (although it can be wrapped into a helper function, if needed):
  ```js
const totalNrOfAssessments = readabilityAssessments.length;
  ```